ts/components/StatusMessages.tsx

```diff
@@ -1,17 +1,14 @@
-import {
-  AlertEdgeToEdgeProps,
-  AlertEdgeToEdgeWrapper
-} from "@pagopa/io-app-design-system";
+import { AlertEdgeToEdgeWrapper } from "@pagopa/io-app-design-system";
 import { PropsWithChildren } from "react";
 import { useStatusAlertProps } from "../hooks/useStatusAlertProps";
 
 type StatusMessagesProps = PropsWithChildren;
 
 export const StatusMessages = ({ children }: StatusMessagesProps) => {
-  const alertProps: AlertEdgeToEdgeProps | undefined = useStatusAlertProps();
+  const statusAlert = useStatusAlertProps();
 
   return (
-    <AlertEdgeToEdgeWrapper alertProps={alertProps}>
+    <AlertEdgeToEdgeWrapper alertProps={statusAlert?.alertProps}>
       {children}
```

Comment:
The BS does not work. I think we should add the `bottomSheet` here, under the children.